File: 1700-number-of-students-unable-to-eat-lunch/1700-number-of-students-unable-to-eat-lunch.java
class Solution {
    public int countStudents(int[] students, int[] sandwiches) {
        int counter[]=new int[2];
        for (int i:students){
            counter[i]+=1;
        }
        int left=sandwiches.length;
        for (int j:sandwiches){
            if (counter[j]==0){
                break;
            }
            if (left==0){
                break;
            }
            counter[j]-=1;
            left-=1;
        }
        return left;
    }
}

File: 1700-number-of-students-unable-to-eat-lunch/1700-number-of-students-unable-to-eat-lunch.py
class Solution:
    def countStudents(self, students: List[int], sandwiches: List[int]) -> int:
        counter=[0,0]
        for i in students:
            counter[i]+=1
        
        left=len(sandwiches)
        for j in sandwiches:
            if counter[j]==0:
                break
            if left==0:
                break
            counter[j]-=1
            left-=1
        return left
    
    